packages/sources/svelte/src/components/overlays/VtmnSnackbar/README.md

```diff
@@ -0,0 +1,47 @@
+# Quick start
+
+You need to add `@vtmn/svelte` on your `package.json`
+
+```
+npm install @vtmn/svelte
+```
+
+Now, you can use `VtmnSnackbar` on your app.
+
+# How it works
+
+First, you need to place the `VtmnSnackbar` component preferably at your
```

Comment:
First, you need to place the `VtmnSnackbar` component on the top of your application